packages/pkp-base/src/lib/pkp-base.ts

```diff
@@ -235,38 +235,42 @@ export class PKPBase<T = PKPBaseDefaultParams> {
       throw new Error('controllerSessionSigs must be an object');
     }
 
-    // If authSig is not provided but sessionSigs are, use the first sessionSig as authSig. In executeJs, the sessionSigs will take priority.
-    let authSig = this.controllerAuthSig;
-    if (
-      !authSig &&
-      this.controllerSessionSigs &&
-      Object.values(this.controllerSessionSigs).length > 0
-    ) {
-      authSig = Object.values(
-        this.controllerSessionSigs
-      )[0] as unknown as AuthSig;
-    }
-
-    if (!authSig) {
-      return this.throwError('authSig is required');
-    }
-
-    const executeJsArgs: ExecuteJsProps = {
-      ...(this.litActionCode && { code: this.litActionCode }),
-      ...(this.litActionIPFS && { ipfsId: this.litActionIPFS }),
-      authSig: authSig,
```

Comment:
I removed the complex condition in favor of a single condition check for both `controllerAuthSig` and `controllerSessionSigs` being defined. since the validation is not being used in this file I didn't bring it in and added to the other property checks to assert that one or the other but not both is defined.